server/py/services/alerts/crud/alerts.py

```diff
@@ -253,25 +253,26 @@ def _handle_notification(
         keep_cache = True
         active = False
         state["count"] += 1
-        logger.debug("Sending notifications for alert", name=alert.name)
```

Comment:
This log line can be quite useful for debugging. Without it we don't have an indication that notifications are sent for a specific alert.
It should stay.